addons/mail/static/tests/composer/suggested_recipients.test.js

```diff
@@ -46,35 +56,45 @@ test("with 3 or less suggested recipients: no 'show more' button", async () => {
     await contains("button", { count: 0, text: "Show more" });
 });
 
-test("Opening full composer in 'send message' mode should copy selected suggested recipients", async (assert) => {
+test("Opening full composer in 'send message' mode should copy selected suggested recipients", async () => {
     const pyEnv = await startServer();
     const partnerId = pyEnv["res.partner"].create({
-        display_name: "John Jane",
+        name: "John Jane",
         email: "john@jane.be",
     });
     const fakeId = pyEnv["res.fake"].create({
         email_cc: "john@test.be",
         phone: "123456789",
         partner_ids: [partnerId],
     });
-    const { env } = await start();
-    await openFormView("res.fake", fakeId);
-    const def = makeDeferred();
-    patchWithCleanup(env.services.action, {
-        doAction(action) {
-            step("do-action");
-            assert.strictEqual(action.name, "Compose Email");
-            assert.strictEqual(action.context.default_subtype_xmlid, "mail.mt_comment");
-            assert.strictEqual(action.context.default_partner_ids.length, 2);
-            const johnTestPartnerId = pyEnv["res.partner"].search([
-                ["email", "=", "john@test.be"],
-                ["phone", "=", "123456789"],
-            ])[0];
-            assert.deepEqual(action.context.default_partner_ids, [johnTestPartnerId, partnerId]);
-            def.resolve();
-            return Promise.resolve();
-        },
+    const def = new Deferred();
+    mockService("action", () => {
+        const ogService = actionService.start(getMockEnv());
+        return {
+            ...ogService,
+            doAction(action) {
+                if (action?.res_model !== "res.fake") {
+                    step("do-action");
+                    expect(action.name).toBe("Compose Email");
+                    expect(action.context.default_subtype_xmlid).toBe("mail.mt_comment");
+                    expect(action.context.default_partner_ids.length).toBe(2);
+                    const johnTestPartnerId = pyEnv["res.partner"].search([
+                        ["email", "=", "john@test.be"],
+                        // ["phone", "=", "123456789"],
```

Comment:
commented instead of removed?

Reply:
Test had some assertions on phone, I removed them because field definition of phone is problematic (computed / related field IIRC), and this looks out-of-scope of this PR, so I removed this part but kept the comment.

I'm removing this comment to officially remove `phone` concerns of the test.

Reply:
But phone is important from test, we want to make sure it is correctly filled.

It doesn't need to be part of search if it helps, but as an extra assert on the found partner.